packages/skill-api/src/core/services/stripe-checkout.ts

```diff
@@ -244,6 +263,30 @@ export async function stripeCheckout({
         },
       })
 
+      const result = LoadedProductSchema.safeParse(loadedProduct)
```

Comment:
check for these values toward the top of the stripe checkout builder so that we can bail early with a better error message if they are missing (missing meaning that some aspect of the `Product -> MerchantProduct -> MerchantPrice` data setup is invalid/inactive.